spec/factories/courses.rb

```diff
@@ -1,4 +1,11 @@
 FactoryGirl.define do
+  factory :user do 
+    first_name "John"
+     last_name "Smith"
+     email "js@email.com"
+     password "password"
+  end 
+
```

Comment:
Create a different file for a factory user. This is a courses. Much like you wouldn't add the `User` class to the file that has the `Course` class. Also, try using the faker gem for these values